resources/js/components/VTSpriteSelect.vue

```diff
@@ -90,6 +90,13 @@ export default {
         pickedSprite = vm.sprites[0];
       }
 
+      if (pickedSprite.name === "Load Custom Sprite") {
+        vm.$emit('load-custom-sprite', true);
+        return;
+      } else {
+        vm.$emit('load-custom-sprite', false);
+      }
```

Comment:
I believe the else here is unnecessary:

```js
      if (pickedSprite.name === "Load Custom Sprite") {
        vm.$emit('load-custom-sprite', true);
        return;
      }
      vm.$emit('load-custom-sprite', false);
```

Reply:
I believe it could just be

`vm.$emit('load-custom-sprite', pickedSprite.name === "Load Custom Sprite");`

Reply:
Reducing it to one line would lose the early return, but getting rid of the unnecessary else seems fine; done.

Reply:
Good point.  I didn't think about the early return, so the if block is still required.